include/aws/mqtt/private/v5/mqtt3_to_mqtt5_adapter_impl.h

```diff
@@ -29,6 +29,11 @@ struct aws_mqtt3_to_mqtt5_adapter_publish_options {
     void *on_complete_userdata;
 };
 
+/*
+ * A subscribe with no subscriptions represents a re-subscribe of all internally tracked topics.  While this
```

Comment:
Us being the only ones to ever directly run across this, this seems fine. Especially since it's well documented what's going on for future poor souls digging around!